index1.js

```diff
@@ -0,0 +1,276 @@
+            if (key === 'ShiftRight') isShiftKeyboard = 2;
+            if (isShift === 0) {
+                isShift = 1;
+                changeCase();
+            }
+        }
+        if (key === 'Shift') {
+            if (isCtrl === 1) {
+                changeLang();
+                isCtrl = 0;
+            } else if (isAlt === 1) {
+                changeLang();
+                isAlt = 0;
+            } else {
+                changeCase();
+                isShift = 1;
+            }
+        }
+        if (key === 'Tab') {
+            textarea.value += '   ';
+        }
+        if (key === 'Backspace') {
+            textarea.value = textarea.value.substr(0, textarea.value.length - 1);
+        }
+        if (key === 'CapsLock') {
+            caps_check = document.querySelectorAll('.caps_check');
+            if (isCaps === 0) {
+                caps_check.forEach(element => element.classList.add('active'));
+                isCaps = 1;
+            } else {
+                isCaps = 0;
+                caps_check.forEach(element => element.classList.remove('active'));
+            }
+            changeCase();
+        }
+        if (key === 'Enter') {
+            textarea.value += '\n';
+            changeLang();
+        }
+
+        if (key === 'ControlRight' || key === 'ControlLeft' || key === 'Ctrl') {
+            isCtrl = 1;
+        }
+        if (key === 'AltRight' || key === 'AltLeft' || key === 'Alt') {
+            isAlt = 1;
+        }
+        
+    }
+
+    function changeCase() {
+        let keyS = document.querySelectorAll(".On");
+        let activeChild = 0,
+            notActiveChild = 0;
+        if (keyS[0].children[0].className === 'show') {
+            activeChild = 0;
+            notActiveChild = 1;
+        } else {
+            activeChild = 1;
+            notActiveChild = 0;
+        }
+        keyS.forEach(function (element) {
+            element.children[activeChild].classList.remove('show');
+            element.children[activeChild].classList.add('hide');
+            element.children[notActiveChild].classList.add('show');
+            element.children[notActiveChild].classList.remove('hide')
+        });
+    }
+
+    function changeLang(place) {
+        let keyS = document.querySelectorAll(".key");
+        let activeChild, notActiveChild;
+        if (keyS[0].children[0].classList[1] === 'On') {
+            activeChild = 0;
+            notActiveChild = 1;
+        } else {
+            activeChild = 1;
+            notActiveChild = 0;
+        }
+        keyS.forEach(function (element) {
+            element.children[activeChild].classList.remove('On');
+            element.children[activeChild].classList.add('Off');
+            element.children[notActiveChild].classList.add('On');
+            element.children[notActiveChild].classList.remove('Off')
+        });
+        if (place === 'start') {} else {
+            if (isCaps) changeCase();
+            if (lang === 'en') {
```

Comment:
все строки нужно выносить в константы. Константы пишутся с большой буквы.
К примеру,
```suggestion
            if (lang === LANGUAGE.EN) {
```